clang/lib/CodeGen/CGSYCLRuntime.cpp

```diff
@@ -20,6 +20,8 @@
 using namespace clang;
 using namespace CodeGen;
 
+namespace {
```

Comment:
I get the purpose here, but this seems to be an unrelated change.  Please submit it as a separate aptch.

Reply:
The problem of absent anonymous namespace (and exported symbol names clash) is triggered by this patch. Separate patch would make it hard to understand the real purpose.

Besides, all patches that I've already submitted are ~5% of the overall line count of the feature. I simply can't split all tiny unrelated changes into separate patches given the time I have.

Reply:
The schedule of your feature doesn't alter our code review guidelines.

Reply:
Still this.

Reply:
I'm not asking to change code review guidelines, I'm providing context. Guidelines are quite flexible, and they are just guidelines. There maybe other factors like business needs.

In this particular case, you probably missed the first part of my reasoning:
> The problem of absent anonymous namespace (and exported symbol names clash) is triggered by this patch. Separate patch would make it hard to understand the real purpose.

what do you think?

Reply:
I did not miss that.